src/tests/facebookOauth.test.js

```diff
@@ -0,0 +1,62 @@
+import chai from 'chai';
+import http from 'chai-http';
+import app from '../app';
+import { mock } from "sinon";
+import 'dotenv/config';
+import {facebookFunction , user} from  '../controllers/facebookOauth'
+
+chai.use(http);
+const { expect } = chai;
+
+const mockUser = {
+    id: 1,
+    firstName: "test@gmail.com",
+    lastName: "test2",
+    roleId:5
+  };
+
+describe("LOGIN WITH FACEBOOK",() =>{
+
+    it('should user login with facebook account', async() =>{
+
+        const mockProvider = 'facebook';
+        const res = await chai.request(app).get(`/auth/${mockProvider}`);
+
+      expect(res.status).to.equal(200);
+
```

Comment:
You are not mocking the Facebook API, this request hits Facebook. Try running your tests with the internet turned off and see these tests failing, meaning they have to hit the Facebook app